src/uct/ib/efa/srd/srd_iface.c

```diff
@@ -352,11 +393,130 @@ uct_srd_iface_poll_tx(uct_srd_iface_t *iface)
     return num_wcs;
 }
 
+static void
+uct_srd_ep_process_rx_desc(uct_srd_iface_t *iface, uct_srd_recv_desc_t *desc)
+{
+    uct_ib_iface_invoke_am_desc(&iface->super, desc->hdr->id, desc->hdr->data,
```

Comment:
maybe jusst
```suggestion
    uct_ib_iface_invoke_am_desc(&iface->super, desc->hdr->id, desc->hdr + 1,
```
and no need to have data in hdr?

Reply:
ok for me fixed